llama_stack/providers/inline/inference/sentence_transformers/sentence_transformers.py

```diff
@@ -43,7 +43,7 @@ async def initialize(self) -> None:
     async def shutdown(self) -> None:
         pass
 
-    async def register_model(self, model: Model) -> None:
+    async def register_model(self, model: Model) -> Model:
```

Comment:
Does this need corresponding SDK changes?

Reply:
I think it's just a reflection of the actual API definition, which is:

```
    async def get_model(self, identifier: str) -> Model: ...
```